spec/support/page_objects/publish/providers_show.rb

```diff
@@ -11,6 +11,15 @@ class ProvidersShow < PageObjects::Base
       element :users, ".app-status-box"
       element :accredited_courses, '[data-qa="courses-as-an-accredited-body"]'
       element :allocations, '[data-qa="allocations"]'
+
+      # The new elements introduced with the new publish navigation are below. When the feature flag is removed the
+      # duplicated elements above can be removed.
+
+      element :nav_organisation_details, :link, "Organisation details"
```

Comment:
We can do, but can we do it when we get rid of the feature flag as `ProvidersShow < PageObjects::Base` will become obsolete. We could maybe turn that into `PrimaryNav`?

I'm not a huge fan of all the abstraction that comes with using page objects, they make the tests much harder to read and write. Do we gain much ROI from using them?

Reply:
Given that these are navigational links and appear everywhere on the site, i think it would make sense that they're contained in their own class. The problem with listing the nav links on this provider show page is that it's incorrect, they're not just owned by this page.

Re: page objects - i mean it's just Ruby and modelling parts of the application so tests can be more focused (and easier) to write. We can have a discussion about it but it's an approach that's becoming more standardised across DFE repos I think

Reply:
That makes sense - thanks. I've pushed up some new commits. ♻️